### PR TITLE
Multilingual support sub folders

### PR DESCRIPTION
**Pull request by mauricio-florez**
This PR pretends to implement multilingual support based on sub-paths `www.domain.com/<country-language>/...`

You can find this strategy working on this link https://mauricio-florez-rsf-altitude-storefront-default.layer0-limelight.link/fr-CA/s/8a6fd51d-9033-475e-9f32-9c57757d4369

Locales supported : [en-CA, fr-CA]

Additionally, hreflangs were also added

### CODE REVIEW

File: components/Link.js [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+        </a>
+      </NextLink>
+    )
+  } else {
+    const child = React.Children.only(children)
+    const passHref = !child.props.href // we only passHref if the child does not have an href prop already.  This fixes a bug with AMP where NextLink overrides and escapes amp-bind syntax in hrefs.
+
+    // This way we can get a ref of Material-ui components
+    return (
+      <RootRef rootRef={ref}>
```
Comment (lissettdm) [inline components/Link.js:76]: Are we keeping `material-ui`?

Comment (mauricio-florez) [reply components/Link.js:76]: we'll remove it in the future however at this point many components rely on this if we remove the rootRef the prefetch functionality would not work properly. Keep in mind link component wraps other child components.

File: components/Link.js [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+            onClick: handleClick,
+            ...other,
+          })}
+        </NextLink>
+      </RootRef>
+    )
+  }
+}
+
+Link.propTypes = {
```
Comment (alejandrosobko) [inline components/Link.js:88]: why this if you can use typescript?

Comment (mauricio-florez) [reply components/Link.js:88]: good point Ale, this is only a copy of the react-storefront native Link component. This component was slightly modified but my intention was not to migrate it to TS, I'm only probing if we can re-use those components and adapt them to whatever we need.

**Review by alejandrosobko — APPROVED** [review]
I tested locally and it's working

**Pull request merged**